app/api/Files.scala

```diff
@@ -725,7 +732,9 @@ class Files @Inject()(
       "size" -> file.length.toString,
       "thumbnail" -> file.thumbnail_id.orNull,
       "authorId" -> file.author.id.stringify,
-      "status" -> file.status)
+      "status" -> file.status,
```

Comment:
A separate request from a user, can you add the following as well:
```suggestion
      "views" -> file.stats.views,
      "downloads" -> file.stats.downloads,
      "status" -> file.status,
```